content/kubernetes/re-clusters/upgrade-redis-cluster.md

```diff
@@ -78,62 +83,96 @@ customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redis
 
-The rolling upgrade of the cluster nodes' statefulSet starts.
+After the operator upgrade is complete, you can upgrade Redis Enterprise cluster (REC).
 
-To see the status of the current rolling upgrade, run:
+### Edit `redisEnterpriseImageSpec` in the REC spec
 
-```sh
-kubectl rollout status sts <REC_name>
-```
+1. Edit the REC custom resource YAML file.
+
+    ```sh
+    kubectl edit rec <your-rec.yaml>
+    ```
+
+1. Replace the `image:` declaration under `redisEnterpriseImageSpec` with the new version tag.
+
+    ```YAML
+    spec:
+      redisEnterpriseImageSpec:
+        imagePullPolicy:  IfNotPresent
+        repository:       redislabs/redis
+        versionTag:       <new-version-tag>
+    ```
 
-## How does the upgrade work?
+1. Save the changes to apply.
 
-Rolling updates allow you to update deployments with zero downtime
-by incrementally updating Redis Enterprise Cluster instances in the pods with new instances.
+### Monitor the upgrade
 
-These diagrams show how a rolling update works:
+You can view the state of the REC with `kubectl get rec`.
 
-- Each hexagon represents a node
-- Each box represents a pod
+  During the upgrade, the state should be `Upgrade`.
+  When the upgrade is complete and the cluster is ready to use, the state will change to `Running`.
+  If the state is `InvalidUpgrade`, there is an error (usually relating to configuration) in the upgrade.
+
+```sh
+$ kubectl get rec
+NAME   NODES   VERSION      STATE     SPEC STATUS   LICENSE STATE   SHARDS LIMIT   LICENSE EXPIRATION DATE   AGE
+rec    3       6.2.10-107   Upgrade   Valid         Valid           4              2022-07-16T13:59:00Z      92m
+```
+
+To see the status of the current rolling upgrade, run:
+
+```sh
+kubectl rollout status sts <REC_name>
+```
 
-![kubernetes-rolling-updates](/images/rs/kubernetes-rolling-updates.png)
+### Upgrade databases
 
-The pods are updated one by one in the diagram, starting from left to right.
-Each pod is updated after the last one completes successfully.
+After the cluster is upgraded, you can upgrade your databases. The process for upgrading databases is the same for both Kubernetes and non-Kubernetes deployments. More details on how to [upgrade a database]({{<relref "/rs/installing-upgrading/upgrading#upgrade-a-database">}}) in the [Upgrade an existing Redis Enterprise Software deployment]({{<relref "/rs/installing-upgrading/upgrading.md">}}) documentation.
 
-![kubernetes-rolling-updates-newapp](/images/rs/kubernetes-rolling-updates-newapp.png)
+Note that if your cluster [`redisUpgradePolicy]({{<relref "/kubernetes/reference/cluster-options#redisupgradepolicy">}}) or your database [`redisVersion`]({{<relref "">}}) are set to `major`, you won't be able to upgrade those databases to minor versions. See [Redis upgrade policy]({{<relref "/rs/installing-upgrading/upgrading#redis-upgrade-policy">}}) for more details.
```

Comment:
This doesn't render correctly, possibly missing the closing backtick on ``[`redisUpgradePolicy]``